Automattic/jetpack
# Added a decoding mechanism to module names and descriptions.

Fixes #5088  .

#### Changes proposed in this Pull Request:
- Adds a decoding library from Calypso.
- Runs module names and descriptions through that on initial state setting.

#### Testing instructions:
- Switch your site language to French.
- Make sure no module names and descriptions have entities like `&rsquo;` in the text.
